i18n/en-US.yml

```diff
@@ -25,87 +25,174 @@ _name: English
 # Component-specific messages (e.g. button captions)
 # are defined for each component under the 'components' category.
 components:
+  AccountSetupFinishPane:
+    message: You are ready to start planning your trips.
+  AfterSignInScreen:
+    mainTitle: Redirecting...
+    message: "If the page doesn't load after a few seconds, <redirectLink>click here</redirectLink>."
+  AmenitiesPanel:
+    noPRLotsFound: No nearby park and ride lots found.
+    # Use plurals to differentiate between company variable being set or not
+    bikesAtStation: "{companyLength, plural,
```

Comment:
One thing to note with this is we lose some formatting ability in our multiline strings. For example, the reportEmailTemplate message produces a syntax error with folded block style in its current form and so I removed some of the space formatting to avoid this.